src/kernels/jupyter/jupyterKernelService.node.ts

```diff
@@ -241,8 +241,7 @@ export class JupyterKernelService implements IJupyterKernelService {
         interpreter: PythonEnvironment,
         kernel: IJupyterKernelSpec,
         specFile: string,
-        cancelToken?: CancellationToken,
-        forceWrite?: boolean
+        cancelToken?: CancellationToken
```

Comment:
Never passed, hence removing this argument (just complicates code with more unnecessary logic)